easybuild/easyconfigs/n/netCDF-Fortran/netCDF-Fortran-4.4.3-foss-2016a.eb

```diff
@@ -0,0 +1,20 @@
+name = 'netCDF-Fortran'
+version = '4.4.3'
+
+homepage = 'http://www.unidata.ucar.edu/software/netcdf/'
+description = """NetCDF (network Common Data Form) is a set of software libraries 
+ and machine-independent data formats that support the creation, access, and sharing of array-oriented 
+ scientific data."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
```

Comment:
this should be changed to `sources = ['v%(version)s.tar.gz']` if the source URL is changed to GitHub